yt_dlp/extractor/crtvg.py

```diff
@@ -0,0 +1,32 @@
+from .common import InfoExtractor
+
+
+class CrtvgIE(InfoExtractor):
+    _VALID_URL = r'https?://www\.crtvg\.es/tvg/a-carta/[^/]+-(?P<id>\d+)'
```

Comment:
If the page loads in browser without the subdomain, we want to make it optional in the `_VALID_URL`.
We also want to make sure we're not matching a fragment (`#`) or query (`?`)
```suggestion
    _VALID_URL = r'https?://(?:www\.)?crtvg\.es/tvg/a-carta/[^/#?]+-(?P<id>\d+)'
```